layout/password.liquid

```diff
@@ -74,7 +74,6 @@
         --text-boxes-shadow-vertical-offset: {{ settings.text_boxes_shadow_vertical_offset }}px;
         --text-boxes-shadow-blur-radius: {{ settings.text_boxes_shadow_blur_radius }}px;
 
-        --buttons-padding: {{ settings.buttons_padding }}px;
```

Comment:
I like the idea of removing buttons / input padding - it will ensure accessible buttons and inputs.  I'm trying to think if there would be a valid art direction for very narrow buttons or inputs, but I think an opinion on function over form makes sense for controls like this.

Reply:
We are confident that the initial button spacing was scalable for most brands. We are insuring to let merchants have a well balanced label over the button background to grab buyers attention. We'll monitor though if that becomes a common request.

There are some tweaks to make to buttons though with the flexibility settings, coming in another issue!